db/patch68.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE `talk_types_link` (
```

Comment:
Slides is a form of talk type. Though agreed the name could be better. Any suggestions ?